bin/test-workflow/QUICKSTART.md

```diff
@@ -0,0 +1,161 @@
+# Conjur Kubernetes Authentication Quick Start Guide
+
+The following guide details an end-to-end workflow for using Conjur Kubernetes
+authentication by deploying a PetStore demo application to a
+Kubernetes-in-Docker (kind) cluster.
+
+At the end of this quickstart, you will have:
+
+* Deployed Conjur OSS to a kind cluster
+* Prepared the cluster with Conjur Config Cluster Prep Helm chart
+* Prepared and enabled the Kubernetes authenticator in Conjur
+* Prepared PetStore app namespace with Conjur NameSpace Prep Helm chart
+* Deployed and verified the PetStore demo application with authentication sidecar
+
+## Prerequisites
+
+* [Docker](https://docs.docker.com/get-docker/) and [kind](https://github.com/kubernetes-sigs/kind#installation-and-usage)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [Helm](https://helm.sh/docs/intro/install/)
+
+# Steps
+
+### Prepare a Cluster and Deploy Conjur OSS
+
+Preparing for the rest of the workflow requires:
+
+* Starting a kind cluster with local Docker registry
+* Creating a new namespace for Conjur, `conjur-oss`
+* Deploying Conjur OSS with Helm
+* Enable the Kubernetes Authenticator in Conjur
+
+The Conjur OSS Helm Chart is published on
+[GitHub](https://github.com/cyberark/conjur-oss-helm-chart).
+This step wraps a Conjur OSS kind deployment
+[example](https://github.com/cyberark/conjur-oss-helm-chart/tree/main/examples/kubernetes-in-docker)
+found there.
```

Comment:
I think it might be useful to rephrase this to provide more detail about the example folder and call out the scripts.

```suggestion
The Conjur OSS Helm Chart repository contains an [example](https://github.com/cyberark/conjur-oss-helm-chart/tree/main/examples/kubernetes-in-docker) folder with scripts and instructions for deploying the Conjur OSS Helm Chart on KinD. The scripts from the example folder are used to accomplish this step by git cloning the Conjur OSS Helm Chart repository and using them to carry out the tasks mentioned above.
```

Reply:
I think we should also explicitly mention the need for git, can't be too careful.

Reply:
- Added detail to mention of Conjur OSS Helm Chart example
- Added `git` to list of prerequisites